pkg/log/tracelog.go

```diff
@@ -0,0 +1,72 @@
+package log
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/uptrace/opentelemetry-go-extra/otelzap"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var (
+	_traceLogger = otelzap.New(zap.NewNop())
+)
+
+// Ctx is a wrapper of otelzap.Ctx, returns a logger with context.
+func Ctx(ctx context.Context) otelzap.LoggerWithCtx {
```

Comment:
The name `Ctx` may be misleading. Using `T(ctx)` may be a better alternative as it denotes tracing, and is consistent with the `L()` and `S()` methods in log pkg.